signalfx-org-metrics/metrics.yaml

```diff
@@ -368,6 +368,16 @@ sf.org.abortedDetectors:
     or data points, whichever caused the detector to stop
   metric_type: cumulative
   title: sf.org.abortedDetectors
+
+sf.org.cingest.grossDpmContentBytesReceived: NBED
```

Comment:
invalid yaml

```suggestion
sf.org.cingest.grossDpmContentBytesReceived:
```

Reply:
@rmfitzpatrick - I know! NBED is a marker I have to easily locate my WIP. I was waiting for more details on the definitions :)